### DISCUSSION TITLE
1+ Hour to build a model with 5 images??

### DISCUSSION THREAD

**Discussion by ZeroCool22** [glucauze/sd-webui-faceswaplab#108] [Q&A]
![Screenshot_6](https://github.com/glucauze/sd-webui-faceswaplab/assets/13344308/1eeafc3e-613d-4b00-bc29-2e51fed40404)

GPU: 1080 TI (11 VRAM)
CPU: 5900x

It's normal to take so long lol?

**Comment by glucauze** [comment]
No, unless your picture is really huge and you are using inpainting. With GPU accel it takes less than a few seconds and on CPU it should takes less than a few dozen.
